x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/elasticsearch/MultilingualE5SmallInternalServiceSettings.java

```diff
@@ -106,4 +107,9 @@ public void writeTo(StreamOutput out) throws IOException {
     public Integer dimensions() {
         return 384;
     }
+
+    @Override
+    public DenseVectorFieldMapper.ElementType elementType() {
```

Comment:
Should we persist this in a new field called `element_type` for all text embedding services?